### ISSUE TITLE
Create language key entry from text selection

### ISSUE THREAD

**Issue by johnwarde** [smallp/vscode-ci#82]
### TLDR;

Automatically create a new language file entry from the currently selected text, ask for a key name and replace the selected text with the appropriate PHP/CodeIgniter code, referring to the entered key name.

### User experience journey

- The user selects a text in the editor, let's call it `<SelectedText>`.
- Invoke command named "CI: Create language key entry from text selection" (for example).
- Ask the editor user for the key name (in a popup) they want to use in the language file, let's call it `<KeyName>`.
- If `<SelectedText>` contains single or double quotes at the start and end of the selection, then 
- - Create a new entry/line in the default or English language file (`$['<KeyName>'] = <SelectedText>;`), save the file automatically.
- - The selected text in the original editor file is replaced with `$this->lang->line('<KeyName>')` i.e. in a PHP context
- If `<SelectedText>` does not contain single or double quotes at the start and end of the selection, then 
- - Create a new entry/line in the default or English language file (`$['<KeyName>'] = '<SelectedText>';`), save the file automatically.
- - The selected text in the original editor file is replaced with `<?php echo $this->lang->line('<KeyName>'); ?>` i.e. in a HTML context

### Benefits
- The developer saves 2 to 3 minutes per string that needs to be translated, there can be a lot of strings in a project.
- Encourages developer to create language ready code from the start of a new project

**Comment by smallp** [comment]
I know you demand. But that is beyond the extension. It's better to create a new extension to quickly handle the language key and value.
I can give you some advise if you want to do that.
